react/transforms/utils.js

```diff
@@ -0,0 +1,3 @@
+export function removeNonWords(string) {
+  return string.replace(/\W/, '')
```

Comment:
This replaces only the first occurrence of a non-word character in `string`.

```js
'abc123!@#'.replace(/\W/, '') // returns "abc123@#"
'abc123!@#'.replace(/\W/g, '') // returns "abc123"
```
```suggestion
  return string.replace(/\W/g, '')
```

Reply:
gotcha. thanks!